packages/i18n/README.md

```diff
@@ -107,10 +107,6 @@ _Parameters_
 Returns a formatted string. If an error occurs in applying the format, the
 original format string is returned.
 
-_Related_
-
--   <http://www.diveintojavascript.com/projects/javascript-sprintf>
```

Comment:
Maybe we should just link to https://www.npmjs.com/package/sprintf-js instead? Because that's what this package bundles.

Reply:
```import sprintfjs from 'sprintf-js';```

This is indeed the same package. I am swapping the link to that.